stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -419,6 +419,68 @@ public Token createTokenSynchronous(final Card card, String publishableKey)
     }
 
     /**
+     * Retrieve an existing {@link Source} on the server. This is useful for checking the status
+     * of a source requiring redirect authentication, such as 3DS cards. Note that this is a
```

Comment:
Some tweaks:
- "on the server" sounds a bit funny; "from the Stripe API" might be better (or you could just link to the docs for the API method we're wrapping: https://stripe.com/docs/api#retrieve_source).
- I think you can remove this sentence: `This is useful for checking the status of a source requiring redirect authentication, such as 3DS cards`, since this method could have other uses, and we're going to be suggesting that people use our polling method to check the status of a source.